Errors from certificate configuration example in guidebook

The example at [](https://quinn-rs.github.io/quinn/quinn/certificate.html#configuring-certificates) doesn't seem to compile. 

Problem one is just an extra parenthesis: 
```rust
let mut crypto = rustls::ClientConfig::builder()
    .with_safe_defaults()
    .with_single_cert(vec![cert]), key)?  // <---- [cert]) should just be [cert]
    .with_no_client_auth();
let client_config = ClientConfig::new(Arc::new(crypto));
```

Fixing that leads to the following error:

```rust
error[E0599]: no method named `with_single_cert` found for struct `ConfigBuilder<rustls::ClientConfig, WantsVerifier>` in the current scope
  --> networking\src\lib.rs:30:10
   |
30 |         .with_single_cert(vec![cert], key)?
   |          ^^^^^^^^^^^^^^^^ method not found in `ConfigBuilder<rustls::ClientConfig, WantsVerifier>`
   |
   = note: the method was found for
           - `ConfigBuilder<rustls::ClientConfig, WantsTransparencyPolicyOrClientCert>`
           - `ConfigBuilder<rustls::ClientConfig, WantsClientCert>`
           - `ConfigBuilder<rustls::ServerConfig, WantsServerCert>`
```

Imo, it could also be clearer that this example isn't really following from the LetsEncrypt docs immediately above this section.

--- a/docs/book/src/quinn/certificate.md
+++ b/docs/book/src/quinn/certificate.md
@@ -129,14 +129,10 @@ This is the only thing you need to do for your server to be secured.
 **Configure Client**
 
 ```rust
-let mut crypto = rustls::ClientConfig::builder()
-    .with_safe_defaults()
-    .with_single_cert(vec![cert]), key)?
-    .with_no_client_auth();
-let client_config = ClientConfig::new(Arc::new(crypto));
+let client_config = ClientConfig::with_native_roots();
 ```
 
-This is the only thing you need to do for your client to trust a server certificate. 
+This is the only thing you need to do for your client to trust a non-self-signed server certificate. 
 
 <br><hr>
 

--- a/docs/book/src/quinn/certificate.md
+++ b/docs/book/src/quinn/certificate.md
@@ -132,7 +132,7 @@ This is the only thing you need to do for your server to be secured.
 let client_config = ClientConfig::with_native_roots();
 ```
 
-This is the only thing you need to do for your client to trust a non-self-signed server certificate. 
+This is the only thing you need to do for your client to trust a server certificate signed by a conventional certificate authority. 
 
 <br><hr>
 

